README.md

```diff
@@ -1,8 +1,53 @@
-Telemetry Services
-------------------
+# Mozilla Pipeline Schemas
 
-This repository contains schemas for Mozilla Firefox Telemetry pings.
+This repository contains schemas for Mozilla's data ingestion pipeline and data
+lake outputs.
 
-These schemas are used to validate incoming submissions at ingestion time. The [RapidJSON](http://rapidjson.org/md_doc_schema.html) library is used for JSON Schema Validation. This has implications for what kinds of string patterns are supported, see the `Conformance` section in the linked document for further details.
+The JSON schemas are used to validate incoming submissions at ingestion time.
+The [RapidJSON](http://rapidjson.org/md_doc_schema.html) library is used for
+JSON Schema Validation. This has implications for what kinds of string patterns
+are supported, see the `Conformance` section in the linked document for further
+details.
 
-To learn more about writing JSON Schemas, [Understanding JSON Schema](https://spacetelescope.github.io/understanding-json-schema/index.html) is a great resource.
+To learn more about writing JSON Schemas,
+[Understanding JSON Schema](https://spacetelescope.github.io/understanding-json-schema/index.html)
+is a great resource.
+
+The [Parquet-MR](https://github.com/apache/parquet-format/blob/master/LogicalTypes.md)
+schemas are used for direct to parquet output.
+
+To learn more about Parquet-MR Schemas.
+[Parquet Schema Examples](https://mozilla-services.github.io/lua_sandbox_extensions/parquet/io_modules/lpeg/parquet.html)
+is a limited resource as the full specification is only documented in code.
+
+## Build
+
+### Note
+
+All schemas are generated from the 'templates' directory and written into the
+'schemas' directory (i.e., the artifacts are generated/saved back into the
+repository as per mreid's request).
```

Comment:
I think we should specify the rationale for this (even if not everyone agrees with it?), rather than just making it sound like we're doing this based on one person's arbitrary decision. :)